client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -388,23 +388,23 @@ export const getCustomsErrors = (
 	};
 };
 
-export const getRatesErrors = ( { values: selectedRates, available: allRates } ) => {
-	return {
-		server: mapValues( allRates, rate => {
-			if ( ! rate.errors ) {
-				return;
-			}
-
-			return rate.errors.map(
-				error =>
-					error.userMessage ||
-					error.message ||
-					translate( "We couldn't get a rate for this package, please try again." )
+export const getRatesErrors = ( { values: selectedRates, available: allRates } ) =>
+	mapValues( allRates, ( rate, boxId ) => {
+		if ( ! isEmpty( rate.errors ) ) {
+			return (
+				rate.errors[ 0 ].userMessage ||
```

Comment:
Can you elaborate a bit? Where are we handling multiple server errors?

Reply:
Here: https://github.com/Automattic/wp-calypso/pull/26979/files#diff-8f6bc9adbfa669876e5a156a4c4411cdL86

We were displaying all the errors returned from the server for each package. I removed it because it seemed like overkill, but it may have a reason to be here.

Reply:
AFAIK this logic is to handle errors coming from multiple rates requests per package (one for each carrier in our current architecture). See commit `795cbca1e181476c8f3bde75e782f0828c5efe45` in the server repo.

Given this, I think we should keep the client-side logic until the server-side changes.

Reply:
I'm good with the test being removed after re-reading. 👍